docs/observability-deployment.md

```diff
@@ -1 +1,189 @@
+
+
+Using a Service Monitor would be the desired approach and will provide your microservice environment with greater inter-operability as your environment scales and evolves.
+
+
+With regards to the _legacy approach_ the Open Liberty operator offers a configuration value to easily instrument the annotations. You can enable Prometheus to begin scraping metrics data from the Open Liberty `/metrics` endpoint by implementing the following snippet into your Open Liberty Operator YAML configuration file.
+
+
+```
+  monitoring:
+    enabled: true
+```
+See https://github.com/OpenLiberty/open-liberty-operator/blob/master/deploy/crds/full_cr.yaml for full template of available fields.
+
+
+### Visualizing your data with Grafana
+
+
+There are IBM provided Grafana dashboards that leverage the metrics tracked from the JVM provided information as well as the Open Liberty runtime.  Details regarding how to deploy and configure Grafana is covered in the following document https://github.com/kabanero-io/guide-app-monitoring#deploy-grafana
+
+You can find out the access point of Grafana by running the following:
+
+```
+# oc get routes -n grafana
+NAME          HOST/PORT                                      PATH      SERVICES      PORT      TERMINATION   WILDCARD
+grafana-ocp   grafana-ocp-grafana.apps.9.37.135.153.nip.io             grafana-ocp   <all>     reencrypt     None
+```
+The `grafana` value is the namespace that you used to deploy Grafana to.
+
+An Open Liberty server configured with MicroProfile Metrics 1.1 will be instrumented with the `mpMerics-1.1` feature in the server's `server.xml`.  Similarly a MicroProfile Metrics 2.0  configured Open Liberty server will be instrumented with the `mpMetrics-2.0` feature. Find the appropriate dashboards at:
+https://github.com/OpenLiberty/open-liberty-operator/deploy/dashboards/metrics
+
+
+## How to use health info with service orchestrator 
+
+
+MicroProfile Health Check allows services to report their health, and it publishes the overall health status to defined endpoints. If a service reports UP, then it’s available. If the service reports DOWN, then it’s unavailable. MicroProfile Health reports an individual service status at the endpoint and indicates the overall status as UP if all the services are UP. A service orchestrator can then use the health statuses to make decisions.
+ 
+### MicroProfile Health Check 1.0
+1.    Follow the instructions from the ci.docker GitHub README https://github.com/WASdev/ci.docker
```

Comment:
should state something here about how this will add mpHealth-1.0 feature to your server's configuration

Reply:
fixed